README.md

```diff
@@ -189,6 +189,12 @@ cfg_condition = "mycond"
     name = "baseline-position"
     version = "3.10"
     ignore = true
+    [[object.property]]
+    name = "events"
+    # generate only `connect_property_events_notify`, without `get_property_events` and `set_property_events`
+    # supported values: "get", "set", "notify",
+    # also supported union "get|set" and inverting "not(notify)"
```

Comment:
Why so complicated that we even need nom? :)

I would simply list the things that should be generated. What is the use-case of negations?

Reply:
For 3 only cases `not` really unusable, I was do it automatically.
Maybe better only support union with simple list splitting and parsing.
I also thought that we need other generate flags in future (for signal), but seems I mixed up with something.

Reply:
Yes better just have a list :)